src/Common/Core/Header/AssetCollection.php

```diff
@@ -32,22 +32,37 @@ public function add(Asset $asset, bool $minify = true): void
     }
 
     /**
+     * @param bool $orderAssets
      * @return Asset[]
      */
-    public function getAssets(): array
+    public function getAssets($orderAssets = false): array
     {
+        if ($orderAssets) {
+            $this->orderAssets();
+        }
+
         return $this->assets;
     }
 
-    public function parse(BaseTwigTemplate $template, string $key): void
+    private function orderAssets(): void
     {
         usort(
             $this->assets,
             function (Asset $asset1, Asset $asset2) {
-                return $asset1->getPriority()->compare($asset2->getPriority());
+                $comparison = $asset1->getPriority()->compare($asset2->getPriority());
```

Comment:
Fixed in https://github.com/jeroendesloovere/forkcms/commit/b1c006cd6b07b2b723df6f42f00921af2e0e16c7